adr/0004-webscraping.md

```diff
@@ -1,5 +1,5 @@
 # 4. Webscraping
-
+gs
```

Comment:
?

Reply:
haha oops that's me thinking I'm in my terminal 😬